components/CreateSpot.js

```diff
@@ -0,0 +1,214 @@
+            />
+            <label for="weathersuggestion">bei mäßigem Wetter</label>
+          </div>
+          <div>
+            <input
+              id="bad_weather"
+              name="bad_weather"
+              type="checkbox"
+              value="bei schlechtem Wetter"
+            />
+            <label for="weathersuggestion">bei schlechtem Wetter</label>
+          </div>
+        </Checkbox>
+        <Checkbox>
+          <Legend>Altersempfehlung</Legend>
+          <div>
+            <input
+              id="children_under_1"
+              name="children_under_1"
+              type="checkbox"
+              value="Kinder unter 1 Jahr"
+            />
+            <label for="agesuggestion">Kinder unter 1 Jahr</label>
+          </div>
+          <div>
+            <input
+              id="children_1_to_3"
+              name="children_1_to_3"
+              type="checkbox"
+              value="Kinder 1 - 3 Jahre"
+            />
+            <label for="agesuggestion">Kinder 1 - 3 Jahre</label>
+          </div>
+          <div>
+            <input
+              id="children_3_to_6"
+              name="children_3_to_6"
+              type="checkbox"
+              value="Kinder 3 - 6 Jahre"
+            />
+            <label for="agesuggestion">Kinder 3 - 6 Jahre</label>
+          </div>
+          <div>
+            <input
+              id="children_6_to_9"
+              name="children_6_to_9"
+              type="checkbox"
+              value="Kinder 6 - 9 Jahre"
+            />
+            <label for="agesuggestion">Kinder 6 - 9 Jahre</label>
+          </div>
+          <div>
+            <input
+              id="children_9_to_12"
+              name="children_9_to_12"
+              type="checkbox"
+              value="Kinder 9 - 12 Jahre"
+            />
+            <label for="agesuggestion">Kinder 9 - 12 Jahre</label>
+          </div>
+          <div>
+            <input
+              id="teenager"
+              name="teenager"
+              type="checkbox"
+              value="Teenager"
+            />
+            <label for="agesuggestion">Teenager</label>
+          </div>
+        </Checkbox>
+        <FormLabels for="tags">Tags</FormLabels>
+        <input id="tags" name="tags" type="text" />
+        <FormLabels for="information">Weitere Infos:</FormLabels>
+        <input id="information" name="information" type="text" />
+        <Button type="submit" variant="submit">
+          Spot erstellen
+        </Button>
+      </CreateForm>
+    </CreateSection>
+  );
+}
+
+const CreateForm = styled.form`
+  display: flex;
+  flex-direction: column;
+`;
+
+const CreateSection = styled.section`
+  background-color: white;
+  border: 1px solid black;
+  padding: 20px;
+  margin: 20px;
+  border-radius: 10px;
+`;
+
+const Checkbox = styled.fieldset`
+  padding: 10px;
+  margin: 10px 0 10px 0;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  border: 1px solid black;
+`;
```

Comment:
I recommend to choose the name a little more descriptive, because checkbox is a little bit misleading. Maybe `OptionsField` it's not the best solution 😁

```suggestion
const OptionsField= styled.fieldset`
  padding: 10px;
  margin: 10px 0;
  display: flex;
  flex-direction: column;
  align-items: flex-start;
  border: 1px solid black;
`;
```